.github/workflows/build_and_test.yml

```diff
@@ -451,9 +451,12 @@ jobs:
         else
           echo "Detected tests " $tests
         fi
-        echo tests="$tests" >> "$GITHUB_OUTPUT"
+
+        echo 'tests<<EOF' >> $GITHUB_OUTPUT
+        echo "$tests" >> "$GITHUB_OUTPUT"
+        echo 'EOF' >> $GITHUB_OUTPUT
   test-flakyness:
-    if: ${{ needs.test-flakyness-pre.outputs.tests != ''}}
+    if: false
```

Comment:
In general it would have been good to have these kind of changes be separate (preferably cherry-picked) commits. That would have made the history easier to follow on the release branches. But I think it's acceptable like this too.